dll/win32/msvcrt/msvcrt.spec

```diff
@@ -199,9 +199,9 @@
 @ cdecl -arch=i386 _CItanh()
 @ stub -version=0x600+ _CrtCheckMemory
 @ stub -version=0x600+ _CrtDbgBreak
-@ stub -version=0x600+ _CrtDbgReport
+@ cdecl -debug-only _CrtDbgReport(long str long str str)
```

Comment:
I've just checked, these functions don't seem to exist in the debug version of the Win2k3 msvcrt.
They are however present on NT6+.

Reply:
> They are however present on NT6+.

![image](https://github.com/reactos/reactos/assets/2107452/a7e7697f-3384-40b4-bf23-6f493eeb4b18)
Indeed. Okay, I will revert.

Reply:
Done.